app/views/legislation/processes/show.html.erb

```diff
@@ -0,0 +1,13 @@
+<% provide :title do %><%= @process.title %><% end %>
+
+<%= render 'legislation/processes/header', process: @process, header: :full %>
+
+<%= render 'documents/additional_documents', documents: @process.documents %>
+
+<%= render 'key_dates', process: @process, phase: :debate_phase %>
+
+<div class="row">
+  <div class="small-12 medium-9 column">
+    <%= @process.homepage.html_safe %>
```

Comment:
In order to remove tags which aren't allowed, how about:
```suggestion
    <%= AdminWYSIWYGSanitizer.new.sanitize(@process.homepage)  %>
```

Reply:
If the homepage is not going to contain tables or so, it's ok